packages/std/src/types.rs

```diff
@@ -104,7 +104,7 @@ pub struct MessageInfo {
     /// is executed such that the new balance is visible during contract execution.
     pub funds: Vec<Coin>,
 }
-
+///Stores information about a smart contract, like its creator, admin, and other relevant details.
```

Comment:
Except it does not. Currently, it only contains the address. We might add more in the future, but for now, you have to query things like creator, admin, etc. using `WasmQuery::ContractInfo`